test/karma-files.json

```diff
@@ -89,6 +90,7 @@
 	"samples/data/usdeur.js",
 	"samples/data/ohlc.js",
 	"samples/data/three-series-1000-points.js",
-	"samples/data/large-dataset.js"
+	"samples/data/large-dataset.js",
 
+	"code/dashboard.src.js"
```

Comment:
Q: Shouldn't be loaded just one?

Reply:
I have a problem with that because it does not matter in which direction we merge, (`rc` to `dashboards/main` or `dashboards/main` to `rc`) it always overwrites HC files with `dashboards/main`. Any ideas, how can we proceed it? I cannot imagine to get through all files and checking each difference

Reply:
Maybe @pawelfus can simulate after the requests a release procedure for Dashboards and I can do the same for Highcharts. This should give us an idea how many hidden issues we have.

Reply:
I am talking about testing the distribution process. So not just looking at code, but transpiling and bundling it. I will give it a shot.

Reply:
I was not able to repeat the steps to create distributions for Highcharts or Dashboards. So we might need some adjustments later on that part of scripts.